File: solutions/3636-check-balanced-string/solution.py
class Solution(object):
    def isBalanced(self, num):
        """
        :type num: str
        :rtype: bool
        """
        n=len(num)
        a,b=0,0
        for i in range(0,n,2):
            a+=int(num[i])
        for i in range(1,n,2):
            b+=int(num[i])
        return a==b

File: solutions/3636-check-balanced-string/solution.java
class Solution {
    public boolean isBalanced(String num) {
        int a=0,b=0,n=num.length(),i=0;
        for(i=1;i<n;i+=2)
        {
            a+=Integer.parseInt(num.charAt(i)+"");
            b+=Integer.parseInt(num.charAt(i-1)+"");
        }
        if(n%2!=0)
            b+=Integer.parseInt(num.charAt(i-1)+"");
        return a==b;
    }
}
